portfolio/src/main/java/com/google/sps/data/Message.java

```diff
@@ -0,0 +1,29 @@
+package com.google.sps.data;
+
+public class Message {
+    public String comment;
+    private long author;
+    private long timestamp;
+    private String email;
+
+    // Email should be optional
+    public Message(String c, long a, long t, String e){
+        comment=c; 
+        author=a; 
+        timestamp=t; 
+        email=e;
+    }
+
+    public Message(String c, long a, long t){
+        comment=c; 
+        author=a; 
+        timestamp=t; 
+        email=null;
+    }
+
+    public String getComment(){return comment;}
+    public void setComment(String c){comment=c;}
+    public long getAuthor(){return author;}
+    public long getTimestamp(){return timestamp;}
+    public String getEmail(){return email;}
+}
```

Comment:
What do you think about creating a static method called fromEntity(Entity entity) that allows you to create a comment from an Entity object?

See below for the cool things you can do with this method.